corehq/apps/receiverwrapper/tests/test_submissions.py

```diff
@@ -138,6 +142,55 @@ def test_invalid_form_submission_file_extension(self):
             f'</message></OpenRosaResponse>'
         )
 
+    def test_submission_with_only_mobile_supported_content_types(self):
+        image = SimpleUploadedFile("image.abc", b"fake image", content_type="application/octet-stream")
+        response = self._submit('simple_form.xml', attachments={
+            "image.png": image,
+        })
+        self.assertEqual(response.status_code, 201)
```

Comment:
Hey @mjriley 

> I prefer to see the entire expectation in the test name

For tests, I would usually just name what is being tested. The expectations would be in the test. I see other tests in this file follow the same naming pattern.

> Is the name here implying that 'application/octet-stream' is a format exclusive to mobile, or that the content type is the only part of this that is valid (as the extension isn't?)

the name is implying the former. Invalid extension was taken for the test, to test that the format is accepted irrespective of valid/invalid extension.

> It might be helpful to replace it with a constant

cool.